docs/packaging-an-application/template-functions.md

```diff
@@ -104,29 +106,32 @@ env_vars:
   static_val: '{{repl HostPublicIpAddress "DB" "redis" }}'
 ```
 
-## HostPublicIpAddressAll
+## HostPublicIpAddressAll [*](/packaging-an-application/template-functions/#notes)
 ```go
 func HostPublicIpAddressAll(componentName string, containerName string) []string
 ```
 Returns host public IP addresses for all instances of a given Component as an array of strings.
+
 Note: `ContainerExposedPortAll`, `HostPrivateIpAddressAll`, `HostPublicIpAddressAll` are guaranteed to return in the same order
 
-## ContainerExposedPort
+## ContainerExposedPort [*](/packaging-an-application/template-functions/#notes)
 ```go
 func ContainerExposedPort(componentName string, containerName string, internalPort string) string
 ```
 Returns the host's public port mapped to the supplied exposed container port as a string.
+
 ```yml
 env_vars:
 - name: REDIS_PORT
   static_val: '{{repl ContainerExposedPort "DB" "redis" "6379" }}'
 ```
 
-## ContainerExposedPortAll
+## ContainerExposedPortAll [*](/packaging-an-application/template-functions/#notes)
```

Comment:
Instead of using a * everywhere I'd like it if we added a first paragraph that said "References another container, please see the <a>template function note on container sequencing and orchestration</a>"